staging/src/k8s.io/csi-translation-lib/plugins/aws_ebs.go

```diff
@@ -174,6 +174,8 @@ func (t *awsElasticBlockStoreCSITranslator) TranslateCSIPVToInTree(pv *v1.Persis
 		ebsSource.Partition = int32(partValue)
 	}
 
+	removeTopology(pv, AWSEBSTopologyKey)
```

Comment:
Sure, I can remove it for now. Although that will make the this a little inconsistent but I guess it is fine.

Reply:
agree, i don't want to block this PR from my lack of understanding.

It might be simpler to put the bug fix for 2. in a separate PR, then we can discvuss it there without blocking item 1 which is simply to migrate from beta label to ga

Reply:
Yeah I thought about separating this PR into 2. But then I figured if we submit 1 then the migration function will just broken. Hmm, maybe we should do 2 first and then do 1.